### ISSUE TITLE
Feature request: Custom jasper report template

### ISSUE THREAD

**Issue by kokeksibir** [WoozyMasta/archimate-ci-image#10]
Hi! Is it possible to extend functionality to use a custom report template?

Archi cli accepts an argument `--jasper.template` for passing full path to the main.jrxml Jasper Reports template file.

**Comment by WoozyMasta** [comment]
Yes, it's possible, but I don't have time to implement it right now.

Therefore, I will gladly accept PR or wait until there is time to do it.

Might be worth just allowing any additional custom argument to be specified here in https://github.com/WoozyMasta/archimate-ci-image/blob/b38830882aebe5af4b6b85865733aaffd939fff6/entrypoint.sh#L84

**Comment by kokeksibir** [comment]
Hi @WoozyMasta thanks for reply. 

I was going to allow custom arguments by modifying this line:

https://github.com/WoozyMasta/archimate-ci-image/blob/b38830882aebe5af4b6b85865733aaffd939fff6/entrypoint.sh#L45

into this:

`  local -a _args=("${@}")`

Then I realized that when there are arguments provided to entrypoint, it behaves differently:

https://github.com/WoozyMasta/archimate-ci-image/blob/b38830882aebe5af4b6b85865733aaffd939fff6/entrypoint.sh#LL148C1-L153C3

In order not to break backwards compatibility I've decided not to remove these lines.

For now I modified my gitlab_ci file by adding another script call to generate jasper report, it works fine. 
```yaml
  script:
    - /opt/Archi/entrypoint.sh
    - /opt/Archi/entrypoint.sh --modelrepository.loadModel <project path> --jasper.createReport <project path>/public --jasper.format PDF --jasper.filename <project name> --jasper.title <project name> --jasper.template <project path>/report-template/main.jrxml
```
But that means it generates jasper report twice. I wanted to keep jasper report generation bc you inject PDF link to HTML page, which is cool. 

Any suggestions?

**Comment by WoozyMasta** [comment]
@kokeksibir
Okay, I'll try to find some time this weekend and try to implement it.

But I'm not promising anything yet.

**Comment by WoozyMasta** [comment]
in 5.7.0-1.0.6

```
with:
  archiJasperReportEnabled: 'true'
  archiJasperReportFormats: 'PDF,DOCX'
  archiExtraArgs: >-
    --jasper.template /path/report-template/main.jrxml
    --jasper.title My_Project
```
